web/frontend/src/components/slam.vue

```diff
@@ -56,17 +56,14 @@ const fetchSLAMMap = (name: string): Promise<Uint8Array> => {
       const chunk = res.getPointCloudPcdChunk_asU8();
       chunks.push(chunk);
     });
-    getPointCloudMap.on('status', (status: { code: number, details: string, metadata: string }) => {
-      if (status.code !== 0) {
-        const error = {
-          message: status.details,
-          code: status.code,
-          metadata: status.metadata,
-        };
+    getPointCloudMap.on('status', (status?) => {
```

Comment:
1. what does it mean if `status` is undefined?
2. why is this changing?

Reply:
this is no longer changing

Reply:
repeated [my smoke test from before](https://github.com/viamrobotics/rdk/pull/2189#discussion_r1170319829) and saw the same behavior after reverting this change.